### DISCUSSION TITLE
Track unlogged users

### DISCUSSION THREAD

**Discussion by Pixelshaped** [plausible/analytics#1922] [Anything Else]
Hey!

On some pages I'd like to be able to separate audiences between connected / not-connected users. I'm not sure I can achieve that using URL parameters (as Plausible only supports a few). URL is the same (say "app/") so I can't rely on that either. Adding an event to all those pages seems a bit redundant with the pageview.

How would you do it? Thanks!

**Comment by metmarkosaric** [comment]
thanks for the feedback @Pixelshaped! we're currently working on allowing you to send custom dimensions with page views. this should help your use case. you can learn more here #320 and see the work in progress here https://github.com/plausible/analytics/pull/1900. i cannot make any promise on when this will go live but do stay tuned!

**Comment by metmarkosaric** [comment]
We're now opened up a beta version of this to everyone. You can attach custom properties (also known as custom dimensions in Google Analytics) when sending a pageview to Plausible in order to create custom metrics such as logged in or logged out. Instructions at https://plausible.io/docs/custom-pageview-props
